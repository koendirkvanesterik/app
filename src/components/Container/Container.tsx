import classNames from 'classnames'
import React from 'react'

export const Container: React.FunctionComponent = ({ children }) => (
  <div className={classNames(['max-w-screen-2xl', 'mx-auto', 'px-10'])}>{children}</div>
)
