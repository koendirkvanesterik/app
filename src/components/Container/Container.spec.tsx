import React from 'react'

import { render } from '@testing-library/react'

import { Container } from './Container'

it('should render render as expected', () => {
  const { container } = render(<Container />)
  expect(container).toMatchSnapshot()
})
