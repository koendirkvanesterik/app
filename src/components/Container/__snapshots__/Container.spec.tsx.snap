// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render render as expected 1`] = `
<div>
  <div
    class="max-w-screen-2xl mx-auto px-5 lg:px-10"
  />
</div>
`;
